py3status/modules/insync.py

```diff
@@ -85,14 +87,17 @@ def insync(self):
             }
 
         color = self.py3.COLOR_DEGRADED
+        format = self.format
         if status == "Insync doesn't seem to be running. Start it first.":
             color = self.py3.COLOR_BAD
-            status = STRING_ERROR
+            format = STRING_ERROR
```

Comment:
I've changed it as I realised that formating like "I{number of syncing files}" would turn into I{error message}" which was harder to read. This was intended as a bug fix.